nautobot/docs/installation/nautobot.md

```diff
@@ -221,10 +221,10 @@ $ echo napalm >> $NAUTOBOT_ROOT/local_requirements.txt
 
 By default, Nautobot will use the local filesystem to store uploaded files. To use a remote filesystem, install the [`django-storages`](https://django-storages.readthedocs.io/en/stable/) library and configure your [desired storage backend](../../configuration/optional-settings/#storage_backend) in `nautobot_config.py`.
 
-To use remote file storage, add `django-storages` to your `local_requirements.txt` so that it can be installed and kept up to date:
+To use remote file storage, add `nautobot[mysql]` to your `local_requirements.txt` so that it can be installed and kept up to date:
 
 ```no-highlight
-$ echo django-storages >> $NAUTOBOT_ROOT/local_requirements.txt
+$ echo "nautobot[storages]" >> $NAUTOBOT_ROOT/local_requirements.txt
```

Comment:
Feels weird to me to be putting `nautobot` into `local_requirements.txt` in any way shape or form. Maybe this change should be reverted? Compare to similar verbiage in `authentication/ldap.md` that you haven't updated in this way.

Reply:
I agree, but I think there is a fundamental difference between `pip install django-storages` and `pip install nautobot[remote_storage]` I believe the latter will follow the version requirements specified by the nautobot package which are driven by the `pyproject.toml` when the package is built.  Installing `django-storages` could potentially have version conflicts.  

That said it's still odd adding nautobot in an additional requirements file.  This might bring the larger issue up in that `local_requirements.txt` could potentially drive dependency issues with Nautobot itself.  Should we recommend some kind of poetry environment be setup locally, I'm not sure how that would look exactly...